### ISSUE TITLE
can't replace runner node

### ISSUE THREAD

**Issue by joschi36** [MonolithProjects/ansible-github_actions_runner#45]
I've just used this role to install GitHub Runner. Installing works however replacing doesn't work. 

```
command: "{{ runner_dir }}/config.sh --url {{ github_server }}/{{ github_owner | default(github_account) }} \
         --token {{ registration.json.token }} --name {{ runner_name }} --unattended --replace"
```

```
fatal: [github-runner-01]: FAILED! => {"changed": true, "cmd": ["/opt/actions-runner/config.sh", "--url", "https://github.com/ORGNAME", "--token", "REDACTED123", "--name", "github-runner-01", "--unattended", "--replace"], "delta": "0:00:00.407194", "end": "2020-12-02 12:51:34.586467", "msg": "non-zero return code", "rc": 1, "start": "2020-12-02 12:51:34.179273", "stderr": "Cannot configure the runner because it is already configured. To reconfigure the runner, run 'config.cmd remove' or './config.sh remove' first.", "stderr_lines": ["Cannot configure the runner because it is already configured. To reconfigure the runner, run 'config.cmd remove' or './config.sh remove' first."]}
```
_(removed stdout for better readability.)_

I'm not sure if this command worked in an older version than 2.274.2.

I've installed the latest version (1.4.0) with these parameters:

```yaml
github_account: ORGNAME
runner_user: github-runner
runner_org: true
access_token: REDACTED
```

**Comment by MonolithProjects** [comment]
Hi there, thanks this issue. I can confirm that this BUG is present also in the latest version 1.4.2.
Fix is on the way...

**Comment by MonolithProjects** [comment]
Issue resolved in version [1.4.3](https://github.com/MonolithProjects/ansible-github_actions_runner/releases/tag/1.4.3)

**Comment by joschi36** [comment]
Thank you very much, works now like a charm :smiley:
